brianlong/www.validators.app
# 183882576 leaders daemon for pythnet

#### What's this PR do?
- add pythnet leader daemon

#### How should this be manually tested?
- run `rails r daemons/leader_stats_pythnet_update_daemon.rb`
- check leaders component on the homepage

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/183882576)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
